content/blog/announcing-http-add-on.md

```diff
@@ -0,0 +1,145 @@
+- ***Operator**:  This is an [Operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) that runs for the convenience of the user. It listens for new [CRD](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) resources, called `HTTPScaledObject`s, and creates and configures interceptors and scalers so that your existing [`Deployment`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) will begin autoscaling according to incoming HTTP traffic.
+
+![architecture diagram](/static/img/blog/http-add-on/arch.png)
+
+## Seeing it in action
+
+We’ve aimed to make the HTTP add-on approximately as simple to operate and use as KEDA. 
+
+There are two major steps to using it:
+
+1. Install the KEDA HTTP add-on operator
+2. Create a new `HTTPScaledObject`
+
+### Installing the Operator
+
+This is an optional step, but we highly recommend it because the operator makes interacting with the HTTP add-on more convenient. We’ve packaged the operator up in a [Helm](https://helm.sh) chart, which you can install by following the instructions below.
+
+_The commands below have been tested on Mac and Linux (Ubuntu). They likely work on Windows WSL2, but won’t work on Powershell. You’ll also need to ensure you have access to a Kubernetes cluster with a Kubernetes configuration file properly configured._
+
+#### Install KEDA
+
+First, make sure you’ve set up an environment variable for the namespace you’d like to install everything into:
+
+```shell
+export NAMESPACE=mynamespace
+```
+
+Next, install KEDA:
+
+```shell
+helm repo add kedacore https://kedacore.github.io/charts
+helm repo update
+helm install --create-namespace -n ${NAMESPACE} keda kedacore/keda
+```
+
+#### Install the Operator
+
+Now that KEDA is installed, you can install the operator:
+
+```shell
+helm install --create-namespace -n ${NAMESPACE} http-add-on kedacore/keda-add-ons-http
+```
+
+### Install an `HTTPScaledObject`
+
+The `HTTPScaledObject` CRD instructs the operator to install and configure the interceptor and scaler for a specified `Deployment`. For this to work, you’ll need to have a Deployment running in the same `$NAMESPACE` and a Service configured to route traffic to the pods in that `Deployment`.
+
+>We've created a sample application [here](https://github.com/kedacore/http-add-on/tree/main/examples/xkcd) as a Helm chart for you. It comes complete with a properly configured `HTTPScaledObject`, so when you install the chart, everything will be automatically configured for you and you can skip the rest of this section. If you install this chart, you can skip the rest of this section.
+
+After you have your application set up, copy the following YAML into a file called myautoscaledapp.yaml:
+
+```yaml
+kind: HTTPScaledObject
+apiVersion: http.keda.sh/v1alpha1
+metadata:
+    name: myapp
+spec:
+    scaleTargetRef:
+        deployment: myAppDeployment
+        service: myAppService
+        port: 8080
```

Comment:
Is this up to date?

Reply:
yup! here is the Go code from which the yaml is generated: https://github.com/kedacore/http-add-on/blob/main/operator/api/v1alpha1/httpscaledobject_types.go#L121-L128

Reply:
Just wanted to make sure we are up to date 👍